WordPress/Classes/Categories/NSString+Helpers.m

```diff
@@ -188,4 +188,13 @@ - (NSArray *)tokenize
     return tokens;
 }
 
+- (BOOL)isWordPressComURL
+{
+    NSRegularExpression *protocol = [NSRegularExpression regularExpressionWithPattern:@"wordpress\\.com/?$" options:NSRegularExpressionCaseInsensitive error:nil];
```

Comment:
This regex will check if the string contains a WordPress.com path, but not necessarily that it *is* a valid path.  No real risk the way its used in this PR but I'm wondering about future use cases. Should it also test for other things like protocol, whether there are invalid chars, and if its just the path is a substring nested in a larger phrase? 
Push back if this is overkill :D